mono/mini/aot-runtime.c

```diff
@@ -6010,10 +6010,10 @@ i32_idx_comparer (const void *key, const void *member)
 }
 
 static int
-i16_idx_comparer (const void *key, const void *member)
+ui16_idx_comparer (const void *key, const void *member)
 {
 	int idx1 = GPOINTER_TO_INT (key);
-	int idx2 = *(gint16*)member;
+	int idx2 = *(guint16*)member;
 	return idx1 - idx2;
```

Comment:
This is still a bit hazardous.
You should do:

```
return (idx1 < idx2) ? -1 : (idx1 > idx2) ? 1 : 0
```

to ward of any overflow.
I realize it is uint16 so ok in this case.